microshift_networking/microshift_multiple_networks/microshift-cni-multus.adoc

```diff
@@ -6,7 +6,7 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-In addition to the default OVN-Kubernetes Container Network Interface (CNI) plugin, {microshift-short} uses an implementation of the Multus CNI to chain other CNI plugins.
+In addition to the default OVN-Kubernetes Container Network Interface (CNI) plugin, {microshift-short} uses an implementation of the Multus CNI to chain other CNI plugins. Installing and using {microshift-short} Multus is optional.
```

Comment:
instead of "in, {microshift-short} uses an implementation of the Multus CNI to chain other CNI plugins.", consider, "Multus CNI is available to chain other CNI plugins."